po/de.po

```diff
@@ -580,27 +603,9 @@ msgstr "Technische Details"
 msgid "Send To Mobile Device"
 msgstr "An Mobilgerät senden"
 
-#. Service Menu
-#: src/extension.js:91 src/extension.js:237
-msgid "Mobile Devices"
-msgstr "Mobile Geräte"
-
-#. TRANSLATORS: A menu option to activate the extension
-#: src/extension.js:119 src/extension.js:380
-msgid "Turn On"
-msgstr "Aktivieren"
-
-#: src/extension.js:232
-#, javascript-format
-msgid "%d Connected"
-msgid_plural "%d Connected"
-msgstr[0] "%d verbunden"
-msgstr[1] "%d verbunden"
-
-#. TRANSLATORS: A menu option to deactivate the extension
-#: src/extension.js:377
-msgid "Turn Off"
-msgstr "Deaktivieren"
+#: src/extension.js:46
+msgid "Sync between your devices"
```

Comment:
I didn't find this new string in Crowdin.

Reply:
I've updated the translations on crowdin, can you see it now?

Reply:
Yes, and they are translated now in Crowdin.